<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropMesh</class>
 <widget class="QWidget" name="PropMesh">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>86</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>86</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PropMesh</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&amp;Mesh:</string>
     </property>
     <property name="buddy">
      <cstring>MeshFile</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="Armed::FileButton" name="MeshFile">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>S&amp;keleton:</string>
     </property>
     <property name="buddy">
      <cstring>SkeletonFile</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="Armed::FileButton" name="SkeletonFile">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="DelSkeleton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Remove skeleton</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Play &amp;animation:</string>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <property name="buddy">
      <cstring>Animation</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="Animation"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Armed::FileButton</class>
   <extends>QPushButton</extends>
   <header>FileButton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>MeshFile</tabstop>
  <tabstop>SkeletonFile</tabstop>
  <tabstop>DelSkeleton</tabstop>
  <tabstop>Animation</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
